utils.sh

```diff
@@ -241,7 +241,41 @@ function node_map_to_install_config_hosts() {
           disableCertificateVerification: ${disable_certificate_verification}
         bootMACAddress: ${mac}
         bootMode: ${boot_mode}
+        networkConfig: |
+          routes:
+            config:
+            - destination: 0.0.0.0/0
+              next-hop-address: ${PROVISIONING_HOST_EXTERNAL_IP}
+              next-hop-interface: ${PROVISIONING_NETWORK_INTERFACE}
```

Comment:
We normally would have the default route on the External network, is the change to the provisioning network intentional?